style.css

```diff
@@ -0,0 +1,469 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap');
+@import url('about.css');
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+section {
+  padding-left: 2%;
+  padding-right: 2%;
+}
+
+/* Hamburger button */
+.hamburger {
+  position: absolute;
+  top: 0;
+  left: 10px;
+  z-index: 10;
+  cursor: pointer;
+  width: 20px;
+  height: 20px;
+  background: none;
+  border: none;
+}
+
+.hamburger-top,
+.hamburger-middle,
+.hamburger-bottom {
+  position: absolute;
+  width: 20px;
+  height: 2px;
+  background-color: #272a31;
+  transition: all 0.5s;
+}
+
+.hamburger-middle {
+  transform: translateY(5px);
+}
+
+.hamburger-bottom {
+  transform: translateY(10px);
+}
+
+/* Transform hamburger to X when open */
+.open {
+  transform: rotate(90deg);
+}
+
+.open .hamburger-top {
+  transform: rotate(45deg) translateY(6px) translateX(6px);
+  background-color: #fff;
+}
+
+.open .hamburger-middle {
+  display: none;
+}
+
+.open .hamburger-bottom {
+  transform: rotate(-45deg) translateY(6px) translateX(-6px);
+  background-color: #fff;
+}
+
+/* Mobile Menu */
+.overlay-show {
+  display: block !important;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(13, 13, 13, 0.804);
+  z-index: 3;
+  backdrop-filter: blur(5px);
+}
+
+.mobile-main-menu {
+  position: fixed;
+  width: 70%;
+  height: 100%;
+  z-index: 3;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  transform: translateX(-100%);
+  transition: transform 0.6s cubic-bezier(0.19, 1, 0.22, 1);
+  background-image: url(asset/background-2.png);
+  padding: 30% 6% 0 6%;
+}
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓